src/server.js

```diff
@@ -57,33 +55,28 @@ server.get('/', async(req, res) => {
 server.get('/create', checkAuthenticated, async(req, res) => {
 
     console.log("USER ID IS " + req.user.id)
-    res.render("content-create.ejs", { name: req.user.firstName })
-
+    res.render("create.ejs", { name: req.user.firstName })
 })
 
 server.get('/login', checkNotAuthenticated, (req, res) => {
     res.render('login.ejs')
 })
 server.post('/login', checkNotAuthenticated, passport.authenticate("local", {
 
-    successRedirect: "/homepage",
+    successRedirect: "/create",
```

Comment:
want to keep prev value "/homepage"

Reply:
right forgot those changes were mine, thanks